### DISCUSSION TITLE
Tasmota Timer Problem. Auto unchecking enable?

### DISCUSSION THREAD

**Discussion by CyprusWildlife** [arendst/Tasmota#22463] [Support request]
Hello,
I have recently just set up a simple system with BlitzWolf SHP with Tasmota 14.3.0. All I need is to turn on the power at 6am, and turn it off at 23:45. I have configured wifi, my ntp server and enabled 1 and 2 under configure timer for on and off and it works.

Problem is, every once in a while, it stops working and when I log in to Configure Timer, I see my rules are not enabled anymore. I did a bit of research and already used setoption65 to OFF and setoption36 to 0 but cannot figure out what else may be causing this.

I am currently at a loss to figure out what's causing my timer enable checkbox to go unchecked on their own. Sometimes its one of them, sometimes its both, and I do not seem to be losing any other config. Any ideas?

![Screenshot 2024-11-10 112948](https://github.com/user-attachments/assets/ae8a8997-c713-45b1-9de6-d422cef7abad)

**Comment by barbudor** [comment]
I can't see any obvious reason for such behavior
I would advise you setup a syslog receiver to store logs for a long time and enable a level 3 loglevel

**Comment by j-schick** [comment]
Hello,

I do have the same problem with several NOUS A1T. 
I have configured 4 Timer to turn on/off on certain events (time, sunset, sunrise) but from time to time, the enable flag of the timers in use disappears while the remaining parameters are still remain the same.
I noticed this behaviour with V14.3.0 as well as with V13.4.0. I made no tests with other release versions so far. 
I have attached a syslog with level 3, wich starts with "Enable":1 for the timers and after a while, 2 of the timers changed to "Enable":0 
[syslog.txt](https://github.com/user-attachments/files/17991587/syslog.txt)

Best regards, Jürgen

**Comment by arendst** [comment]
What happens if you enable `Repeat`?

If `Repeat` is not set it will disable the timer if it has expired

**Reply by j-schick** [reply]
Jeez, I totally missed that. Thank you for the help

**Comment by maxuser74** [comment]
Same problem with 14.4.1, Enable flag behave unexpectedly and changes its state autonomously.
Repeat flag is always turned on.
Now, I have set up Setoption65 to OFF and Setoption36 to 0, let's see if something changes! (...I honestly don't remember the previous setting of these two options).
BTW, not a single issue with previous versions.

**Reply by barbudor** [reply]
Those options have nothing to do with timers
If so65 was triggered the whole tasmota configuration would be erased, not just timers
So36 doesn't disable timers neither, only what are listed which could create problem at boot time.

I believe you have read the answers above and have checked 'repeat'? 🤔

**Reply by maxuser74** [reply]
Ok thanks for the clarification.
Repeat is checked, but "Enable" trigger behave autonomously and unexpectedly.

**Reply by barbudor** [reply]
As my crystal ball is in maintenance, would you kindly elaborate such as providing logs in level 3 when that happens

**Reply by maxuser74** [reply]
I'll do thanks!
